### PR TITLE
Add scUSDC

### PR DESCRIPTION
**Pull request by coolhill**
**Description**

Adding scUSDC token.

### CODE REVIEW

**Review by cfluke-cb — DISMISSED** [review]
looks good from base side

**Review by hamdiallam — APPROVED** [review]

**Comment by hamdiallam** [comment]
hey @coolhill ,

The validation script failed because the decimals dont match. Can you address this

File: data/SCUSDC/data.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Sandclock Yield USDC",
+  "symbol": "scUSDC",
+  "decimals": 18,
```
Comment (coolhill) [inline data/SCUSDC/data.json:4]:
```suggestion
  "decimals": 6,
```

**Comment by coolhill** [comment]
@hamdiallam I changed to 6 decimals which is what the mainnet token contract has. Regarding the Base token contract it was created using the [OptimismMintableERC20Factory](https://basescan.org/tx/0x56e14e21b5ee26880022f000fd48828df4de2efaaf49ab7b7a9e25df560228a6) which gave it 18 decimals as a default. Is this an issue?

**Comment by cfluke-cb** [comment]
Hi @coolhill , Yes we will want the Base deployment to have matching decimals. You can start from the OptimismMintableERC20 token, and override the decimals. Here is that contract -https://basescan.org/address/0xf23d369d7471bd9f6487e198723eea023389f1d4#code#F2#L1

**Comment by hamdiallam** [comment]
@coolhill any update on this?

**Pull request closed without merging**